### ISSUE TITLE
Remove product type "Validation Set" 

### ISSUE THREAD

**Issue by gschwend** [linea-it/pz-server#77]
Following suggestions from the PZ group of the LSST commissioning team, please remove the product type named "Validation Set".  The Validation Set should be part of the same data product of its corresponding Training Set, with objects identified by a flag (an extra column). The motivation is to allow users to rearrange the two subsets differently (e.g., for cross-validation).
